packages/cli/scripts/build.ts

```diff
@@ -43,15 +43,15 @@ async function main() {
     stdio: 'inherit',
   });
 
+  const pkg = await readJSON(join(dirRoot, 'package.json'));
+  const dependencies = Object.keys(pkg?.dependencies ?? {});
   // Do the initial `ncc` build
-  console.log();
-  const args = [
-    'ncc',
-    'build',
-    '--external',
-    'update-notifier',
-    'src/index.ts',
-  ];
+  console.log('Dependencies:', dependencies);
```

Comment:
Non-blocking - what if dependencies is empty? Should we still log?

Reply:
Can we update this in a follow-up PR?

Reply:
On one hand, this is a build script and if there are no dependencies, then I'd think we'd want to see that.

On the other hand, `dependencies` will likely never be empty.